harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -546,10 +557,31 @@ def test_degraded_volume(
 @pytest.mark.upgrade
 @pytest.mark.any_nodes
 class TestAnyNodesUpgrade:
+    @pytest.mark.skip_version_if("< v1.2.0", reason="logging is implemented as addon since v1.2.0")
```

Comment:
We can't do `skip` in the setup case, otherwise we will lose the test case in old version.
(as `test_verify_logging_pods` is depends on this, so all `setup` test case should never be skipped.)